File: ITP1/ITP1_5_C.py
def print_chessboard(h, w):
    for col in range(h):
        for row in range(w):
            if col % 2 == 0:
                if row % 2 == 0:
                    print("#", end='')
                else:
                    print(".", end='')
            else:
                if row % 2 == 0:
                    print(".", end='')
                else:
                    print("#", end='')
        print()
    print()

while True:
    h, w = [int(i) for i in input().split(" ")]
    if h == 0 and w == 0: break
    print_chessboard(h, w)

File: ITP1/ITP1_5_C.rb
def print_chessboard(h, w)
  h.times do |i|
    w.times do |j|
      if i % 2 == 0
        print (j % 2 == 0) ? "#" : "."
      else
        print (j % 2 == 0) ? "." : "#"
      end
    end
    puts
  end
  puts
end

while true do
  h, w = gets.split(" ").map{|s| s.to_i}
  break if h == 0 && w == 0
  print_chessboard(h, w)
end
